ragaai_catalyst/tracers/agentic_tracing/tracers/main_tracer.py

```
@@ -65,14 +64,14 @@ def __init__(self, user_detail, auto_instrumentation=None, timeout=120):
         # Add warning flag
         self._warning_shown = False
 
-        BaseTracer.__init__(self, user_detail)
+        #BaseTracer.__init__(self, user_detail)
 
         self.tools: Dict[str, Tool] = {}
```

Ruff (0.11.9)

Undefined name `Tool`

(F821)